src/cms/templates/pages/page_tree_node.html

```diff
@@ -23,7 +23,7 @@
         <span class="collapse-subpages cursor-pointer inline-block align-middle"
               title="{% trans 'Collapse all subpages' %}"
               data-page-id="{{ page.id }}"
-              data-page-children="{{ page|get_children }}">
+              data-page-children="{{ children_in_filter }}">
```

Comment:
```suggestion
              data-page-children="{{ page|get_children }}">
```
As we fixed now the buggy filter, we can apply it now